navigation/CameraFlowNavigator.tsx

```diff
@@ -1,10 +1,10 @@
-import { useNavigation } from "@react-navigation/native";
-import React, { FunctionComponent, ReactElement, useState } from "react";
-import { View } from "../components/Themed";
+import React, { ReactElement, useState } from "react";
 import CameraRecording from "../screens/CameraRecording";
 import LetsStartRecording from "../screens/LetsStartRecording";
 import PrepareFace from "../screens/PrepareFace";
 import { Text } from "../components/Themed";
+import { useMainNavigation } from "../hooks/useMainNavigation";
+import { NavigationScreens } from "./MainNavigationContext";
 
 interface CameraFlowNavigatorProps<T> {
   objects: T[];
```

Comment:
This is just for my own self actualization and not an actual nitpick of the code but are the props parametrized for flexibility so that we can stick camera recording anywhere? (Like after an episode recall screen or for episode creation)

Reply:
Exactly. For episodes, and other stuff if we need it.